integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -152,13 +152,22 @@ run_sonobuoy() {
 }
 
 cleanup() {
-	info "Results directory "${e2e_result_dir}" will not be deleted"
-	info "View results"
-	cat ${e2e_result_dir}/plugins/e2e/results/global/e2e.log
-	info "View sonobuoy status"
-	sonobuoy status
-	# Remove sonobuoy execution pods
-	sonobuoy delete
+	if [ -d "${e2e_result_dir:-}" ]; then
```

Comment:
are you are allowing overwrite here but not in the next line (`log_file="${e2e_result_dir}/plugins/e2e/results/global/e2e.log"`) ?

Reply:
@devimc actually the idea is not to allow overwrite but instead avoid a "unbounded variable" error. If the `sonobuoy run` fails then `e2e_result_dir` doesn't get initialized. To make it more clear, I can initialize it along side the other variables (e.g. `e2e_result_dir=""`). What do you think?

Reply:
@wainersm sounds good to me, thanks